### PR TITLE
AirfRANS 4L/384d — widen golden config (hidden=384, heads=6)

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis

The 4L/256d golden config achieved val_primary/surface_mse = 0.007264 (PR #2727). Widening the hidden dimension from 256 to 384 (and proportionally scaling heads from 4 to 6) increases model capacity per-layer without adding depth. Wider models often generalise better for geometric/field regression tasks like CFD, where the full width of the hidden representation encodes multiple simultaneous flow features. We keep all other golden hyperparameters fixed.

## Instructions

Run the 4L/384d config (increase width while keeping depth fixed at 4):

```bash
cd target/icml2026 && SENPAI_MAX_EPOCHS=9999 python train.py \
  --dataset airfrans --airfrans_task full --optimizer adamw \
  --lr 7e-4 --cosine-t-max 5 --grad-clip 1.0 --weight-decay 1e-2 \
  --no-use-ema --enable-fourier \
  --model-layers 4 --model-hidden-dim 384 --model-heads 6 \
  --epochs 999 \
  --wandb_group airfrans-arch-scaling
```

Key differences from baseline: `--model-hidden-dim 384 --model-heads 6` (were 256 and 4)

## Baseline

Current best (PR #2727, W&B run: ruurxdqs):
- val_primary/surface_mse: **0.007264** (at epoch 40, hit cap at epoch 50)
- External target: **0.0043**
- Baseline W&B run: [ruurxdqs](https://wandb.ai/wandb/senpai/runs/ruurxdqs)
- Reproduce: `cd target/icml2026 && SENPAI_MAX_EPOCHS=9999 python train.py --dataset airfrans --airfrans_task full --optimizer adamw --lr 7e-4 --cosine-t-max 5 --grad-clip 1.0 --weight-decay 1e-2 --no-use-ema --enable-fourier --model-layers 4 --model-hidden-dim 256 --model-heads 4 --epochs 999`

### CODE REVIEW

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2760 because ADVISOR: Closing — 4L architecture is superseded. 3L/256d achieved 0.001479 (PR #2771). Width scaling is now being tested at 3L (ray #2824). Thank you..
